### DISCUSSION TITLE
Multiple time request problem

### DISCUSSION THREAD

**Discussion by shahensaifullah** [nuxt/nuxt#17333] [General]
![image](https://user-images.githubusercontent.com/44544213/177535704-28f4f1fa-35d7-43d0-be4e-5cc68f0309a5.png)
Here I create a authenticate page. but after sending one request, later, useFetch does not call and cannot make another request.
but if I reload the page, the useFetch function can use for once. 
so, how can I make a several post method or get method in my page?

```
console.log("hello world")
    const { data: count } = await useFetch(`${runtimeConfig.public.BACKEND_URL}/api/auth/seller/login/`,{
      method: "post",
      body: state
    })
```
you can see, that the console print "hello world" several times but send a request only one time.
please resolve my issue.

**Comment by Flanker72** [comment]
You can use `refresh` method exported by `useFetch` to trigger data reloading or use `$fetch` instead if you want to fetch data on demand.
`useFetch` and `useAsyncData` may be called in `script setup`, however `$fetch` may be called in your own functions and handlers (like button click, etc). Use the right tool to achieve the best result!
